src/components/banners/Banner.tsx

```diff
@@ -15,6 +15,15 @@ export type BannerProps = {
   message: ReactNode;
   pinned?: boolean;
   intense?: boolean;
+  // activate showMore feature
+  showMore?: boolean;
+  // text for showMore link
+  showMoreLinkText?: ReactNode;
+  // text for showless link
+  showLessLinkText?: ReactNode;
+  // additionalMessage is shown next to message when clicking showMoreLinkText
+  // disappears when clicking showLess link
+  additionalMessage?: ReactNode;
```

Comment:
What do you think about, instead of having a `showMore` boolean, simplifying the API by having the presence of an `additionalMessage` signify that the showMore feature should be on? Then we could remove the unneeded `showMore` prop.

Reply:
@chowington Thank you for your review and comment. 👍 I have reflected your suggestion!